addons/payment/views/payment_templates.xml

```diff
@@ -358,30 +358,30 @@
         </form>
     </template>
 
-    <!-- Expandable payment icon list -->
-    <template id="icon_list" name="Payment Icon List">
-        <ul class="payment_icon_list float-end list-inline" data-max-icons="3">
-            <t t-set="icon_index" t-value="0"/>
-            <t t-set="MAX_ICONS" t-value="3"/>
-            <!-- === Icons === -->
-            <!-- Only shown if in the first 3 icons -->
-            <t t-foreach="provider.payment_icon_ids.filtered(lambda r: r.image_payment_form)" t-as="icon">
-                <li t-attf-class="list-inline-item{{'' if (icon_index &lt; MAX_ICONS) else ' d-none'}}">
-                    <span t-esc="icon.image_payment_form"
+    <!-- Expandable payment method list -->
+    <template id="payment_method_list" name="Payment Method List">
+        <ul class="payment_method_list float-end list-inline" data-max-icons-displayed="3">
+            <t t-set="payment_method_index" t-value="0"/>
+            <t t-set="MAX_ICONS_DISPLAYED" t-value="3"/>
+            <!-- === Payment Methods Icons === -->
+            <!-- Only shown if in the first 3 methods -->
+            <t t-foreach="provider.payment_method_ids.filtered(lambda r: r.image_payment_form)" t-as="payment_method">
+                <li t-attf-class="list-inline-item{{'' if (payment_method_index &lt; MAX_ICONS_DISPLAYED) else ' d-none'}}">
+                    <span t-esc="payment_method.image_payment_form"
```

Comment:
Decide between "payment method icon" and "icon".

Reply:
For me generally, inside, it makes more sense to use just payment icons as that is what we're referring to, but I left the payment method icons list on the top to clearly show what icons are and where they came from.